src/components/suggest/suggest_item.tsx

```diff
@@ -123,14 +150,20 @@ export const EuiSuggestItem: FunctionComponent<EuiSuggestItemProps> = ({
 
   let colorClass = '';
 
-  const labelDisplayClass = classNames(
+  const labelClassNames = classNames(
     'euiSuggestItem__label',
     labelDisplayToClassMap[labelDisplay],
     {
       'euiSuggestItem__labelDisplay--expand': !description,
+      [`euiSuggestItem__labelWidth${labelWidth}`]: labelDisplay === 'fixed',
```

Comment:
To match BEM naming like the description classes, this should actually be

```suggestion
      [`euiSuggestItem__label--width${labelWidth}`]: labelDisplay === 'fixed',
```